vue.js-case-study.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Vue.js Case Study
+author: Lars Halvor Vikse Kallåk <lhvk>
+tags: javascript, framework, vue
+---
+
+## Introduction
+ 
+ ![Image of Vue logo](https://res.cloudinary.com/practicaldev/image/fetch/s--q6AIFazN--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ll22z8mea106b4vjdjy7.jpeg)
+
+I've chosen Vue as my case study because frontend is what I study. I've never used Vue and I have zero experience with it, so I reckon this would be pretty interesting to dive deeper into.
+
+## Brief History
+
+Vue was created by Evan You, a Google software engineer, in 2013 and had its official release in 2014. The name derives from the French word vue, meaning view. Evan wanted to create a framework that was lightweight and included the best parts from Angular, which he had been using at Google.
+
+#### Here is a timeline highlighting the major events:
+ 
+- **2013:** Version 0.6 In December Evan You created the first version of Vue
+- **2014:** Version 0.8 First public release
+- **2015:** Version 1.0 Released
```

Comment:
It might be useful to include how many users they had at various milestones

Reply:
I would also like to know what was happening around 2013 that led Evan to start this project.